pkg/internal/checkup/checkup.go

```diff
@@ -318,9 +318,9 @@ func newTrafficGen(checkupConfig config.Config) *kvcorev1.VirtualMachineInstance
 		ContainerDiskImage:              checkupConfig.TrafficGeneratorImage,
 		NetworkAttachmentDefinitionName: checkupConfig.NetworkAttachmentDefinitionName,
 		NICEastMACAddress:               checkupConfig.TrafficGeneratorEastMacAddress.String(),
-		NICEastPCIAddress:               config.VMIEastNICPCIAddress,
+		NICEastPCIAddress:               config.VMIUnderTestEastNICPCIAddress,
```

Comment:
Currently, the abstract name kind of fits both VMIs.
Now the traffic gen VMI is given inputs with the ``VMIUnderTest` prefix.
This feels wrong.

Reply:
I think that keeping the same guest PCI-Addresses for both VMIs is confusing, and thus should be changed. Following this PR I will introduce the PCI-Addresses for the traffic-gen VMI.